sycl/doc/extensions/supported/sycl_ext_oneapi_usm_device_read_only.asciidoc

```diff
@@ -0,0 +1,100 @@
+= SYCL_INTEL_usm_device_read_only
```

Comment:
Can you use the new [template](https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/template.asciidoc) for this extension specification?  Also, follow the instructions in the [README-process](https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/README-process.md) document.  Sorry, this is relatively new, and I should have pointed it out before.

Reply:
OK